src/components/LeftPane/Drawers/AnalyticsDrawer/AdequacyDonut.tsx

```diff
@@ -0,0 +1,82 @@
+import { Chart, ChartData, ChartTooltipItem } from 'chart.js'
+import 'chart.piecelabel.js'
+import { keyBy, round } from 'lodash'
+import * as React from 'react'
+import { Doughnut } from 'react-chartjs-2'
+import { StoreProps, withStore } from '../../../../services/store'
+import { StatsBox } from '../../../StatsBox/StatsBox'
+
+type Props = StoreProps & {
+  serviceAreas: string[]
+}
+
+/**
+ * Use circular legend patches instead of the default rectangles.
+ *
+ * TODO: Fix typings upstream in DefinitelyTyped/chart.js
+ */
+(Chart as any).defaults.global.legend.labels.usePointStyle = true
+
+export let AdequacyDonut = withStore('adequacies')<Props>(({ serviceAreas, store }) => {
+
+  let serviceAreasHash = keyBy(serviceAreas)
+  let adequacies = store.get('adequacies')
+  let rps = store.get('representativePoints')
+  let rpsInServiceAreas = rps.filter(_ => _.service_area_id in serviceAreasHash)
+  let adequateRpsInServiceAreas = rpsInServiceAreas.filter(_ => adequacies[_.id] === true)
+
+  let numAdequate = adequateRpsInServiceAreas.length
+  let numInadequate = rpsInServiceAreas.length - numAdequate
+  let percentAdequate = round(100 * numAdequate / rpsInServiceAreas.length)
+  let percentInadequate = 100 - percentAdequate
+
+  return <div className='AdequacyDonut'>
+    <Doughnut
```

Comment:
Donut or Doughnut? Both are used in this file

Reply:
Good catch - changed to Doughnut